scripts/js/commands/api/convertApiDocsToHistorical.ts

```diff
@@ -143,12 +144,22 @@ async function generateJsonFiles(
   await writeFile(`${projectNewHistoricalFolder}/_toc.json`, tocFile + "\n");
 }
 
-async function copyImages(pkgName: string, versionWithoutPatch: string) {
+async function copyImages(
+  pkgName: string,
+  hasSeparateReleaseNotes: boolean,
+  versionWithoutPatch: string,
+) {
   console.log("Copying images");
   const imageDirSource = `public/images/api/${pkgName}/`;
   const imageDirDest = `public/images/api/${pkgName}/${versionWithoutPatch}`;
   await mkdirp(imageDirDest);
-  await $`find ${imageDirSource}/* -maxdepth 0 -type f | grep -v "release_notes" | xargs -I {} cp -a {} ${imageDirDest}`;
+
+  // If the project only has a single release notes file, we should not copy the release notes images.
+  if (hasSeparateReleaseNotes) {
+    await $`find ${imageDirSource}/* -maxdepth 0 -type f | xargs -I {} cp -a {} ${imageDirDest}`;
+  } else {
+    await $`find ${imageDirSource}/* -maxdepth 0 -type f | grep -v "release_notes" | xargs -I {} cp -a {} ${imageDirDest}`;
+  }
```

Comment:
I didn't try DRYing this more because I think it can make the escaping confusing with `zx`. Imo it's not that bad to have it duplicated, and we're at low risk of them falling out of sync since the two lines are so close together.

Reply:
I think the input defs for `copyObjectsInv` should include the `hasSeparateReleaseNotes` var correct?

Reply:
Shouldn't be relevant for `objectsInv`. No matter what, we move the `objectsInv` file to be in the new historical folder at `public/api/<pkg>/<version>`. Doesn't matter whether release notes are a single page or one-per-release

Reply:
Ope nvm I misread what these lines are replacing. This looks good to me!